app/views/layouts/application.html.erb

```diff
@@ -28,11 +28,14 @@
 <body>
 <%= render 'shared/skip_links' %>
 <%= render 'shared/header' %>
-<%= render 'shared/alerts' %>
 <%= yield :breadcrumbs %>
 
 <main role="main" id="main">
   <div class='l-constrained'>
+    <div class="l-flashes">
```

Comment:
Sure, thought that might be the case. A demo could be good in the meeting today actually!